packages/http-client/src/index.ts

```diff
@@ -1,18 +1,19 @@
 import fetch from "isomorphic-fetch";
 import type { Includes, Project } from "./index.d";
 
-const protocol = process.env.NODE_ENV === "production" ? "https" : "http";
-
 export const loadProject = async ({
+  apiUrl,
   projectId,
-  host = process.env.DESIGNER_HOST || "localhost:3000",
   include = { tree: true, props: true, breakpoints: true },
 }: {
+  apiUrl: string;
   projectId: string;
-  host?: string;
   include?: Includes<boolean>;
 }): Promise<Project> => {
-  const baseUrl = new URL("/", `${protocol}://${host}`);
+  if (apiUrl === undefined) {
```

Comment:
Again, I mentioned in the CR that if type is not optional, this logic doesn't make sense, what makes sense is validating after new URL the set hostname